internal/internal_workflow.go

```diff
@@ -860,7 +860,12 @@ func getStackTraceRaw(top string, omitTop, omitBottom int) string {
 		return rawStack
 	}
 	lines := strings.Split(rawStack, "\n")
-	lines = lines[omitTop : len(lines)-omitBottom]
+	// If the top is after the end, set as same
+	omitEnd := len(lines) - omitBottom
+	if omitTop > omitEnd {
+		omitTop = omitEnd
```

Comment:
I would rather not omit in this case, so we have a chance to see the full stack trace, what do you think?

Reply:
That can work. This really shouldn't happen ever anyways, so I am a bit disappointed that we're having to decide how to handle the invalid index when it does. But unfortunately without the ability to easily replicate, I can never know I fixed it otherwise.

I will make the change.

Reply:
Done.